x/oracle/abci.go

```diff
@@ -96,15 +96,20 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
 		//---------------------------
 		// Do miss counting & slashing
 		for _, claim := range validatorClaimMap {
-			// Skip abstain & valid voters
 			// we require validator to have submitted in-range data
 			// for all assets to not be counted as a miss
 			if int(claim.WinCount) == totalTargets {
 				continue
 			}
 
+			if int(claim.WinCount+claim.AbstainCount) == totalTargets {
```

Comment:
WinCount is the count of votes that the validator made that are within the 1% range of the weighted median. So, if they win count is equal to total targets, that means the validator was in range on all of the targets, if they specifically abstained on some, we expect them to win the ones they didnt abstain to qualify as `abstaining` instead of `miss`